skills/sql-summary/scripts/save_to_db.py

```
@@ -0,0 +1,61 @@
+        sys.exit(1)
+
+    try:
+        from datus.cli.generation_hooks import GenerationHooks
+
+        result = GenerationHooks._sync_reference_sql_to_db(full_path, agent_config, args.build_mode)
+
+        if result.get("success"):
+            print(json.dumps({"success": True, "message": result.get("message", "Saved successfully")}))
+        else:
+            print(json.dumps({"success": False, "error": result.get("error", "Unknown error")}))
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)